lib/src/block_syntaxes/table_syntax.dart

```diff
@@ -71,29 +72,42 @@ class TableSyntax extends BlockSyntax {
   }
 
   List<String?> _parseAlignments(String line) {
-    final startIndex = _walkPastOpeningPipe(line);
+    final columns = <String?>[];
+    // Set the value to `true` when hitting a non whitespace character other
+    // than the first pipe character.
+    var started = false;
+    var hitDash = false;
+    String? align;
```

Comment:
Let's name this, `alignment`, as a noun.

Reply:
Fixed